packages/auth/src/platform_browser/recaptcha/recaptcha_enterprise_verifier.ts

```diff
@@ -88,13 +107,17 @@ export class RecaptchaEnterpriseVerifier {
       const grecaptcha = _window().grecaptcha;
       if (isEnterprise(grecaptcha)) {
         grecaptcha.enterprise.ready(() => {
-          grecaptcha.enterprise.execute(siteKey, { action: 'login' })
+          try {
+            grecaptcha.enterprise.execute(siteKey, { action })
             .then((token) => {
               resolve(token);
             })
             .catch((error) => {
```

Comment:
This chained .catch is probably redundant from the looks of it

Reply:
Yeah, it looks so, while I just confirmed that the `catch` block associated with `try` won't catch the promise error.